codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -386,8 +395,9 @@ createSchema = do
   fixScopedNameLookupTables
   addProjectTables
   addMostRecentBranchTable
-  execute insertSchemaVersionSql
   addNameLookupMountTables
+  addMostRecentNamespaceTable
+  execute insertSchemaVersionSql
```

Comment:
What's the significance of moving `insertSchemaVersionSql` to the bottom? Is it meant to remain at the bottom?

Reply:
No significance, this all occurs in a transaction and the migrations themselves shouldn't depend on it being any particular value. It just seemed like it belonged more at the bottom than in the middle of the migrations.